addons/mail/models/mail_message.py

```diff
@@ -597,9 +597,9 @@ def _get_with_access(self, message_id, mode="read", **kwargs):
         return self.browse()
 
     @api.model_create_multi
-    def create(self, values_list):
+    def create(self, vals_list):
```

Comment:
Most of changes are useless, could you keep the fixes only ?

Reply:
No, the linter is checking the names of parameters.

For example, calling `model.create(vals_list=[...])` should work, but it does not because of such overrides.